### PR TITLE
bugfix_196

### PR DESCRIPTION
**Pull request by dalisch**
#### Overview
* refactored file resource management by utilizing resource default for file permissions and ownership
* removed 'recurse' property on all files in $rdeck_base and subdirs
* addresses issue https://github.com/voxpupuli/puppet-rundeck/issues/196

### CODE REVIEW

**Comment by jyaworski** [comment]
Please squash.

File: manifests/install.pp [hunk 1]
```diff
@@ -89,35 +95,23 @@
...
-    group   => $group,
-    mode    => '0640',
+  file { $rundeck::params::framework_config['framework.ssh.keypath']:
+    owner => $user,
+    group => $group,
+    mode  => '0600',
   }
 
-  file { $rundeck::params::framework_config['framework.etc.dir']:
+  file { $rundeck::params::service_logs_dir:
```
Comment (jyaworski) [inline manifests/install.pp]: Can you remove ::params and just have rundeck::service_logs_dir?

**Comment by igalic** [comment]
@dalisch you have a 'Merge branch 'master' into refactor_file_permissions'
and that's not really a desired outcome from a [rebase](https://www.git-scm.com/book/en/v2/Git-Branching-Rebasing)

**Comment by danifr** [comment]
+1 to this change

**Comment by dalisch** [comment]
Agreed -- rebasing...

**Comment by dalisch** [comment]
closing this PR, will open a new one...

**Comment by dalisch** [comment]
@igalic , any idea why the master branch in my fork is failing, but is passing in the main repo?
https://travis-ci.org/dalisch/puppet-rundeck/builds/114898671

**Comment by igalic** [comment]
@dalisch we've only merged recently merged the pr that fixes it all: https://github.com/voxpupuli/puppet-rundeck/pull/201
if you rebase now off of master, it should work

**Comment by dalisch** [comment]
thanks @igalic -- (re)-rebasing, PR coming you way...

**Pull request closed without merging**